doc/pages/upgrade/030-upgrade_100_101.md

```diff
@@ -0,0 +1,236 @@
+existent.
+
+All you have to do to overcome this temporary limitation is to edit your Astarte resource by
+explicitly setting the Astarte Flow image you plan to use:
+```yaml
+spec:
+  ...
+  components:
+    ...
+    flow:
+      image: <the-astarte-flow-image>
+```
+
+All the available Astarte Flow's tags can be found
+[here](https://hub.docker.com/r/astarte/astarte_flow/tags?page=1&ordering=last_updated).
+
+## Deploy `AstarteDefaultIngress` in place of `AstarteVoyagerIngress`
+
+The current section describes the procedure for replacing the deprecated `AstarteVoyagerIngress`
+with the new `AstarteDefaultIngress`. If the Voyager ingress is not deployed within your cluster,
+feel free to skip this section.
+
+The first step requires the installation of the [NGINX Ingress
+Controller](https://kubernetes.github.io/ingress-nginx/):
+
+```bash
+$ helm repo add ingress-nginx https://kubernetes.github.io/ingress-nginx
+$ helm repo update
+$ helm install ingress-nginx ingress-nginx/ingress-nginx -n ingress-nginx \
+    --set controller.service.externalTrafficPolicy=Local \
+    --create-namespace
+```
+
+Then, the `AstarteDefaultIngress` must be deployed. If your Kubernetes installation supports
+LoadBalancer ingresses (most managed ones do), you should be able to get away with the most standard
+CR:
+
+```yaml
+apiVersion: ingress.astarte-platform.org/v1alpha1
+kind: AstarteDefaultIngress
+metadata:
+  name: adi
+  namespace: astarte
+spec:
+  ### Astarte Default Ingress CRD
+  astarte: astarte
+  tlsSecret: <your-astarte-tls-secret>
+  api:
+    exposeHousekeeping: true
+  dashboard:
+    deploy: true
+    ssl: true
+    host: <your-astarte-dashboard-host>
+  broker:
+    deploy: true
+    serviceType: LoadBalancer
+```
+
+Further details on this topic can be found at [this page](064-setup_astartedefaultingress.html). If
+your certificate is issued by cert-manager after the solution of an HTTP challenge, please refer to
+[this
+section](064-setup_astartedefaultingress.html#how-to-support-automatic-certificate-renewal-for-http-challenges)
+to support the automatic renewal of your certificate.
+
+When the `AstarteDefaultIngress` resource is created, the Operator is responsible for the creation
+of:
+- an NGINX ingress which is devoted to routing requests to both the Astarte API and the Astarte
+  Dashboard;
+- a service of type LoadBalancer that exposes the Astarte broker to the outer world.
+
+As soon as both the ingress and the broker service have an assigned external IP, update your Cloud
+DNS zone record sets with the new external IPs. Assuming that your Astarte and AstarteDefaultIngress
+are named `astarte` and `adi` respectively and that they resides in the `astarte` namespace, you can
+find the external IPs with:
+
+```bash
+$ # retrieve information about the ingress
+$ kubectl get ingress -n astarte
+NAME              CLASS   HOSTS          ADDRESS   PORTS     AGE
+adi-api-ingress   nginx   <your-hosts>   X.Y.W.Z   80, 443   6s
```

Comment:
`adi-api-ingress` -> `adi-astarte-api-ingress`

Reply:
If the AstarteDefaultIngress is named `adi`, the ingress will be `adi-api-ingress` (see [here](https://github.com/astarte-platform/astarte-kubernetes-operator/blob/973884f968430dd75212622eec35bc65db61b07e/lib/defaultingress/api_ingress.go#L108-L110)).